test-application/parent.fyre

```diff
@@ -20,6 +20,11 @@ def ssr_request():
       <b>
         {get_parent_signal()}
       </b>
+      <ulo>
+        <li>a</li>
+        <li>b</li>
+        <li>c</li>
+      </ulo>
```

Comment:
I don't think that we should keep this. This will make the test-application unrunnable

Reply:
Thanks, for spotting that! I will adjust.